src/test/application/PikoLoggerTest.js

```diff
@@ -0,0 +1,145 @@
+import {JSDOM} from 'jsdom'
+import {expect} from 'chai'
+import sinon from 'sinon'
+import {PikoLogger} from '../../main/application/PikoLogger'
+
+describe('PikoLogger', () => {
```

Comment:
@alextremp  If I export directly PikoLogger class with logger as an static method, then I lose the stored in memory loggers map. 
Maybe because of this I would maintain the initialiazier class that return the Piko instance.